Makefile

```diff
@@ -24,9 +24,7 @@ inplace:
 	$(PYTHON) setup.py build_ext -i
```

Comment:
Maybe add a comment to explain that this is only useful when building with setuptools, e.g. after having run `python setup.py develop` or by setting `PYTHONPATH` manually.

Maybe we could have a `dev-setuptools` entry to make that more explicit? 
Not sure about the latter.